lte/gateway/release/BUILD.bazel

```diff
@@ -229,8 +229,14 @@ pkg_deb(
     package = MAGMA_PKGNAME,
     package_file_name = "{fname}.deb".format(fname = MAGMA_FILE_NAME),
     postinst = ":magma-postinst-bazel",
-    provides = [MAGMA_PKGNAME],
```

Comment:
I think the old `provides` and `replaces` fields were unnecessary. While it's possible to set it, it seems this is not done by upstream Ubuntu packages.

Reply:
confirm: I took this over to the bazel build for consistency - if it's unnecessary, then we can remove it.